arangod/Aql/RestAqlHandler.cpp

```diff
@@ -98,7 +98,7 @@ std::pair<double, std::shared_ptr<VPackBuilder>> RestAqlHandler::getPatchedOptio
   {
     VPackObjectBuilder guard(options.get());
     TRI_ASSERT(optionsSlice.isObject());
-    for (auto const& pair : VPackObjectIterator(optionsSlice)) {
+    for (auto pair : VPackObjectIterator(optionsSlice)) {
```

Comment:
any reason you didn't remove auto here too?

Reply:
VPackObjectIterator::ObjectPair is a tad long